var/spack/repos/builtin/packages/r-rmarkdown/package.py

```diff
@@ -42,3 +43,6 @@ class RRmarkdown(RPackage):
     depends_on('r-evaluate', type=('build', 'run'))
     depends_on('r-base64enc', type=('build', 'run'))
     depends_on('r-jsonlite', type=('build', 'run'))
+    depends_on('r-rprojroot', type=('build', 'run'))
+    depends_on('r-mime', type=('build', 'run'))
+    depends_on('r-stringr', type=('build', 'run'))
```

Comment:
Can you add the version constraints as well:
https://cran.rstudio.com/web/packages/rmarkdown/index.html